### ISSUE TITLE
Sadly doesn't work for me

### ISSUE THREAD

**Issue by Septias** [paytonrules/AirCombat#2]
I'm new to rust/Godot so I just wanted to try out some example projects, so I cloned your project, build it with cargo, and ran it on Godot 3.2.
The game loads fine, but I can't use the buttons to start the game :/
I get this error message:

```
E 0:00:36.928   emit_signal: Error calling method from signal 'pressed': 'Control(TitleScreen.gdns)::_on_newgame_pressed': Method not found..
  <C++-Quellcode>core/object.cpp:1260 @ emit_signal()
```

**Comment by paytonrules** [comment]
What that usually means is that the gdns file isn't wired properly to the corresponding class in the Rust code. Usually - because the name is wrong. This can happen if you rename a class in Rust and don't update the gdns resource. I'll take a look.

**Comment by paytonrules** [comment]
Did you get any errors on build? I am currently running Godot 3.2.3, and was able to run it just fine on the latest godot-rust, and just for safety I cloned a clean version of the repo and it was all fine.

**Comment by paytonrules** [comment]
@Septias Did you close this because you got it fixed? I'd still like to address it if possible.

**Comment by Septias** [comment]
No, I just closed it because I didn't look at it in the past. But since you're interested I'll try it tomorrow.

**Comment by Septias** [comment]
I pulled the newest version and rebuild it. Then I downloaded Godot 3.2.3 and opened the project.
When I click on any Button I get: 
`Error calling method from signal 'pressed': 'Control(TitleScreen.gdns)::_on_quitgame_pressed': Method not found..`
I'm on windows 10.

**Comment by paytonrules** [comment]
Ahhh Windows - I bet that's the problem. Maybe a path somewhere is backwards. I'm on a mac but I do occasionally use a VM for this kind of thing, I can give it a shot.
